code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -85,11 +85,7 @@
 
 /datum/pipeline/proc/addMachineryMember(obj/machinery/atmospherics/components/considered_component)
 	other_atmosmch |= considered_component
-	var/datum/gas_mixture/component_mixture = considered_component.returnPipenetAir(src)
-	if(!component_mixture)
-		stack_trace("addMachineryMember: Null gasmix added to pipeline datum from [considered_component] \
-		which is of type [considered_component.type]. Nearby: ([considered_component.x], [considered_component.y], [considered_component.z])")
-	other_airs |= component_mixture
+	other_airs |= considered_component.returnPipenetAir(src)
```

Comment:
This can still return an empty list can't it?

Reply:
I reimplemented the old stack_trace thing to check the length and blare an error if we get an empty list. Will this suffice?

Reply:
that'll do yeah